lgc/interface/lgc/Pipeline.h

```diff
@@ -81,6 +81,13 @@ enum class NggSubgroupSizing : unsigned {
                     ///  primsPerSubgroup
 };
 
+/// Enumerate denormal override modes.
+enum class DenormalMode : unsigned {
```

Comment:
No sure if you need to pass this option to LGC. Currently, only front-end uses this option.

Reply:
llpcPipelineContext.cpp is based on the lgc types, so I think this is unavoidable?

Reply:
llpcPipelineContext less depends on LGC types now. LGC has its own LgcContext. I am OK if you keep this code.